plugins/postcss-focus-visible/README.md

```diff
@@ -59,10 +59,10 @@ instructions for:
 ### preserve
 
 The `preserve` option determines whether the original notation
-is preserved. By default, it is not preserved.
+is preserved. By default, it is preserved.
```

Comment:
This was actually wrong as per https://github.com/csstools/postcss-plugins/blob/main/plugins/postcss-focus-visible/src/index.ts#L7